### ISSUE TITLE
Content overlapping window

### ISSUE THREAD

**Issue by janvorisek** [AlexTorresSk/custom-electron-titlebar#30]
I am having a problem with heights. In my case the `.container-after-titlebar` is having the same height as `body` and therefore silently overlaps the windows by titlebar height (30px).

I have temporatily solved the issue for me.

```
html,
body {
  width: 100%; // Not needed to work but elements would have zero height
  height: 100%; // Not needed to work but elements would have zero height
}

.container-after-titlebar {
  height: auto !important; // custom-electron-titlebar possible bug!
}
```

Seems like `height: 100%` should not be applied to `.container-after-titlebar`

**Comment by lane-c-wagner** [comment]
This is correct, can confirm I'm seeing the same thing. My apologies, 100% height was not a good solution.

I think a better solution would be for "container-after-titlebar" to have a height of "100% - titlebarHeight" ideally.

**Comment by AlexTorresSk** [comment]
Fixed in #31.

Thanks for your contribution 💯
